### ISSUE TITLE
Put everything in an organized layout

### ISSUE THREAD

**Issue by mm-ansarian** [mm-ansarian/average_calculator#1]
Hi!

I hope you be ok.

There is a problem with this program's GUI; as you know, this project's purpose is calculating the average of several numbers, a program which is usable for anyone; so it should have a beautiful graphical user interface to be easy for user to work with this app.

I designed every text boxes, buttons and text browsers very clean, but when the size of main window change, the other widgets in the main window don't resize.

I tried to create a layout for the main window and I was successful! now the size changing of the main window is synchrenized with the other widgets, but I couldn't restore that beautiful designing in this layout; It seems that the layout of main window wants to force me to design everything based on its commands!

_**In short, I want to have a layout in the main window meanwhile keep the current UI format.**_

You can see the current UI format below:

<img width="450" alt="Screenshot 2024-08-21 011038" src="https://github.com/user-attachments/assets/31999592-3449-461d-900d-2376e7afd84e">
